bayesian/default_config.py

```diff
@@ -49,3 +49,7 @@
 CORE_DEPENDENCIES_REPO_URL = os.getenv('CORE_DEPENDENCIES_REPO', 'https://github.com/'
                                                                  'fabric8-analytics/'
                                                                  'booster-core-dependencies')
+
+GEMINI_SERVER_URL = "http://{host}:{port}".format(
```

Comment:
You can make use of gremlin url from - https://github.com/fabric8-analytics/fabric8-analytics-server/blob/master/bayesian/utils.py#L41

Reply:
I meant, you can also use GEMINI in utils.